.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,3 +34,4 @@
 - [ ] The changes are tested and work correctly
 - [ ] `pre-commit` checks pass / the code has been formatted using Black
 - [ ] Tests have been added to verify that the new code works
+- [ ] Device diagnostics data has been attached.
```

Comment:
nit: slightly inconsistent dot placement 😅 
I'd probably just remove it, but we can also add it to all checklist parts if you prefer.
```suggestion
- [ ] Device diagnostics data has been attached
```